src/signals/settings/categories/components/styled.ts

```diff
@@ -103,6 +103,32 @@ export const DeleteButton = styled(Button)`
   height: 44px;
   width: 44px;
 `
+export const StyledInfo = styled.p`
+  margin-top: ${themeSpacing(0)};
+  margin-bottom: ${themeSpacing(1)};
+`
+
+export const InvisibleButton = styled.button<{ toggle: boolean }>`
+  text-decoration: none;
+  background-color: unset;
+  color: inherit;
+  border: none;
+  padding: 0;
+
```

Comment:
I see this is copies from some other location. I think we should reuse it instead. And maybe call it rotating button instead?

Reply:
Vincent made the reusable component Detail, it's being used in the IconInput. This was a result of a bad clean up.